tests/unit/conftest.py

```diff
@@ -15,16 +16,12 @@ def settings() -> Settings:
 
 
 @fixture(scope="session", autouse=True)
-def redis(settings):
-    """Flush redis before tests."""
-
-    redis_connection = get_redis_connection(
-        url=settings.redis_om_url, decode_responses=True
-    )
-    redis_connection.flushall()
-
+def redis():
+    redis_my_proc = factories.redis_proc()
+    redis_connection = factories.redisdb(redis_my_proc)
```

Comment:
If we can't setup as a url:
```suggestion
    redis_connection = factories.redisdb(redis_my_proc)

    # Clean up if needed
    class Meta
        database = redis_connection
    
    BaseModel.meta.database = redis_connection
```